index.css

```diff
@@ -388,3 +403,213 @@ label {
   font-weight: 400;
   line-height: 28px;
 }
+
+.list_style {
+  list-style: none;
+}
+
+@media only screen and (min-width: 768px) {
+  .nav1 {
+    display: none;
+  }
+
+  .nav2 {
+    display: block;
+    display: flex;
+    gap: 20px;
+  }
+
+  .nav2 a {
+    font-size: 15px;
+    font-weight: 500;
+    color: #344563;
+  }
+
+  .section1 {
+    display: flex;
+    justify-content: center;
+  }
```

Comment:
Please notice that your section isn't filling the whole screen:
![image](https://user-images.githubusercontent.com/42799579/199866891-cac4d7a6-9711-47eb-a550-a915168ab5cc.png)

Kindly adjust its width to make your project stretch to any screen size 😉.

Note: just like you did in the contact section 😄:
![image](https://user-images.githubusercontent.com/42799579/199866955-2da70202-e575-48d1-8f67-255db7b09cb5.png)